lib/http_perform.c

```diff
@@ -318,8 +334,12 @@ sf_bool STDCALL http_perform(CURL *curl,
             } else if (http_code != 200) {
                 retry = is_retryable_http_code(http_code);
                 if (!retry) {
-                    SET_SNOWFLAKE_ERROR(error, SF_STATUS_ERROR_RETRY,
-                                        "Received unretryable http code",
+                    char msg[1024];
```

Comment:
This change is new and is due to some customer cases that we see that get `Received unretryable http code`. I thought it make sense to know the `http_code` to give us a better view of why we are getting this!